Description: Change system rc file location
 Prevent config file conflicts with user Bazel installations.
Author: Yun Peng <pcloudy@google.com>
Origin: upstream, https://salsa.debian.org/bazel-team/bazel-bootstrap/-/commit/3f57d658ec8818e37a1519eee824f13d7ec6a4e2
Forwarded: not-needed
Last-Update: 2020-08-17

--- a/src/main/cpp/BUILD
+++ b/src/main/cpp/BUILD
@@ -171,7 +171,7 @@
         # For posix platforms, this can include environment variables in the
         # form ${var_name}. Braces are required.
         "//conditions:default": [
-            "-DBAZEL_SYSTEM_BAZELRC_PATH=\\\"/etc/bazel.bazelrc\\\"",
+            "-DBAZEL_SYSTEM_BAZELRC_PATH=\\\"/etc/bazel/bazelrc\\\"",
         ],
     }),
     visibility = [
